src/components/NanoContract/NanoContractTransactionActionList.component.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { useSelector } from 'react-redux';
+import { t } from 'ttag';
+import { COLORS } from '../../styles/themes';
+import { NanoContractTransactionActionListItem } from './NanoContractTransactionActionListItem.component';
+import { HathorFlatList } from '../HathorFlatList';
+import { FeedbackContent } from '../FeedbackContent';
+
+const getActions = async () => {
+  // TODO: To implement on wallet-lib
+};
+
+/**
+ * It presents a list of actions of a transaction.
+ *
+ * @param {Object} props
+ * @param {Object} props.tx Transaction data
+ */
+export const NanoContractTransactionActionList = ({ tx }) => {
+  const wallet = useSelector((state) => state.wallet);
+  const [actions, setActions] = useState([]);
+
+  useEffect(() => {
+    // TODO: implement fetchActions to get actions of a transaction
+    const fetchActions = async () => {
+      // TODO: getActions must be implemented in the lib
```

Comment:
And should also be a saga

Will this PR wait until the lib is updated?

Reply:
No. But I can rip off this component until we have the actions on each nano contract transaction.